Modules/TOF/toffull.json

```diff
@@ -78,6 +78,22 @@
           }
         ]
       },
+      "CheckDRMDiagnostics": {
+        "active": "true",
+        "className": "o2::quality_control_modules::tof::CheckDRMDiagnostics",
+        "moduleName": "QcTOF",
+        "detectorName": "TOF",
+        "policy": "OnAny",
+        "dataSource": [
+          {
+            "type": "Task",
+            "name": "TaskRaw",
+            "MOs": [
+              "RDHCounterCrate0"
```

Comment:
This should be the `DRMCounter` no?

Reply:
In the CheckDiagnostics.cxx file it is given as RDH.